src/js/themes/types.ts

```diff
@@ -267,6 +267,16 @@ export interface ThemeColors {
        */
       inverse: string;
     };
+    sepolia: {
+      /**
+       * {string} networks.;sepoliaDefault - For sepolia test network colored elements
```

Comment:
Fix typo. Ah I see could you also fix Goelri as well? 
`networks.;goerliDefault` should be `networks.goerli.default`
```suggestion
       * {string} networks.sepolia.default - For sepolia test network colored elements
```